packages/flutter/test/material/banner_test.dart

```diff
@@ -25,7 +25,7 @@ void main() {
     );
 
     final Container container = _getContainerFromBanner(tester);
-    expect(container.decoration, const BoxDecoration(color: color));
+    expect(container.color, color);
```

Comment:
Here and elsewhere: A simple container with a color no longer creates a BoxDecoration.  This is the break.

Reply:
When making a change like this, are there metrics for before and after? I'm curious about rendering speed (which I would expect to improve) and app size (which I would expect to increase slightly).

Reply:
I recorded but have lost some performance gains in various timeline summary benchmarks we run. The results are not earth shaking, but it is a bump in performance for frame times - we use a lot of containers in the framework, and containers are a very common widget to use.

On the Flutter Gallery, building with this patch is not enough to meaningfully impact the app.so/app.dill sizes when building the release AOT bundle.  They measure using the same number of blocks on my local disk with and without this patch.

Reply:
If I generate the heapsnapshots for each, the one with the ColoredBox takes slightly more space, indicating that the colored box is getting properly shaken out of both types.

The added classes are pretty tiny though :)

Reply:
Thanks for the info. Reading through this, I was suddenly curious what the life cycle of an SDK optimization like this was.